python/pyproject.toml

```diff
@@ -43,6 +43,7 @@ mlflow-skinny = {version = "^1.26.1", optional = true}
 # Pyspark related dependencies
 pyarrow = {version = "^8.0.0", optional = true}
 pyspark = {version = "^3.0.0", optional = true}
+Flask = "^2.1.2"
```

Comment:
will this always install Flask?
Since Flask is needed only for the example, maybe we shouldn't require in the `.toml`, and only ask for the user to install it directly on the example?

Reply:
Good point!